### PR TITLE
Manisha/mar-28

### PR DESCRIPTION
**Pull request by manii283**

### CODE REVIEW

File: resources/views/me/ssc-form.blade.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 @section('main-content')
     @include('me.info-panel')
     <div class="card mb-a8 p-10">
-        <form class="row g-3" method="post" action="/me/ssc" enctype="multipart/form-data">
+        <form class="row g-3" method="post" action="/me/hsc" enctype="multipart/form-data" id='ssc-form'>
```
Comment (ithqindia) [inline resources/views/me/ssc-form.blade.php:7]: ❓

Comment (ithqindia) [reply resources/views/me/ssc-form.blade.php:7]: ssc to hsc

File: resources/views/components/graduation-form.blade.php [hunk 2]
```diff
@@ -145,6 +150,127 @@
...
+                        },
+                    },
+                    graduation_college: {
+                        validators: {
+                            notEmpty: {
+                                message: 'College name is required',
+                            },
+                            stringLength: {
+                                min: 5,
+                                message: 'college name must be more than 5 characters long',
```
Comment (ithqindia) [inline resources/views/components/graduation-form.blade.php:176]: Capital C

File: resources/views/components/graduation-form.blade.php [hunk 3]
```diff
@@ -247,8 +245,7 @@
                                 message: 'Graduation year form is required',
```
Comment (sumedh-mohod) [inline resources/views/components/graduation-form.blade.php:245]: message: year from

File: resources/views/components/personal-information-form.blade.php [hunk 4]
```diff
@@ -50,7 +50,7 @@ class="custom-select form-control @error('salutation') is-invalide @enderror" na
 <div class="row mt-5 md-5">
     <div class="col-12">
         <label for="email" class="form-label">Email<span style="color: red">*</span> </label>
-        <input type="email" class="form-control @error('email') is-invalid @enderror" name="email" id="email"
+        <input type="text" class="form-control @error('email') is-invalid @enderror" name="email" id="email"
```
Comment (ithqindia) [inline resources/views/components/personal-information-form.blade.php:53]: explain this change

File: resources/views/components/personal-information-form.blade.php [hunk 5]
```diff
@@ -234,5 +234,122 @@ class="custom-select form-control @error('salutation') is-invalide @enderror" na
...
 
 <div class="col-12 mt-3">
-<button type="submit" class="btn btn-primary">Submit</button>
+    <button type="submit" class="btn btn-primary" id='submit-btn'>Submit</button>
 </div>
+
+@section('scripts')
+    <script>
+
+let teenager = moment(new Date());
```
Comment (ithqindia) [inline resources/views/components/personal-information-form.blade.php:243]: check formatting - code not aligned

File: resources/views/components/personal-information-form.blade.php [hunk 6]
```diff
@@ -234,5 +234,122 @@ class="custom-select form-control @error('salutation') is-invalide @enderror" na
...
+                        validators: {
+                            notEmpty: {
+                                message: 'The value is not a valid email address',
+                            },
+                        },
+                    },
+                    mobile: {
+                        validators: {
+                            notEmpty: {
+                                message: 'The mobile number is required',
```
Comment (ithqindia) [inline resources/views/components/personal-information-form.blade.php:288]: Change to - Mobile number is required

File: resources/views/components/personal-information-form.blade.php [hunk 7]
```diff
@@ -234,5 +234,122 @@ class="custom-select form-control @error('salutation') is-invalide @enderror" na
...
+                            },
+                        },
+                    },
+                    //  birth_date: {
+                    //      validators: {
+                    //          notEmpty: {
+                    //              message: 'Date of birth is required',
+                    //          },
+                    //      },
+                    // },
```
Comment (ithqindia) [inline resources/views/components/personal-information-form.blade.php:313]: Remove commented code

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: resources/views/components/personal-information-form.blade.php (hunk 4 above)
Comment (manii283) [reply resources/views/components/personal-information-form.blade.php:53]: red border was not applying while validation that's why I was changed

**Review by ithqindia — APPROVED** [review]

**Comment by ithqindia** [comment]
Merged locally

**Pull request closed without merging**